docs/embeddings.md

```diff
@@ -265,13 +342,22 @@ palm_embedding = embedding_functions.GooglePalmEmbeddingFunction(
 
 ```
 
-## HuggingFace
+</TabItem>
+<TabItem value="js" label="JavaScript">
+</TabItem>
 
-Chroma also provides a convenient wrapper around HuggingFace's embedding API. This embedding function runs remotely on HuggingFace's servers, and requires an API key. You can get an API key by signing up for an account at [HuggingFace](https://huggingface.co/).
```

Comment:
This change would help eliminate confusion -- I spent some time trying to figure out how to use this in the JS client before taking a look at the source and noticing it wasn't implemented.

Reply:
good point!